Core/clim-basic/events.lisp

```diff
@@ -188,8 +190,11 @@
 (define-event-class pointer-enter-event    (pointer-boundary-event) ())
 (define-event-class pointer-exit-event     (pointer-boundary-event) ())
 
-(define-event-class pointer-ungrab-enter-event (pointer-enter-event) ())
-(define-event-class pointer-ungrab-leave-event (pointer-exit-event)  ())
+(defclass grab-event () ())
+(define-event-class pointer-grab-enter-event (pointer-enter-event grab-event) ())
+(define-event-class pointer-grab-leave-event (pointer-exit-event grab-event) ())
+(define-event-class pointer-ungrab-enter-event (pointer-enter-event grab-event) ())
+(define-event-class pointer-ungrab-leave-event (pointer-exit-event grab-event) ())
```

Comment:
is there rationale behind adding grab-event class?
please add a comment describing what these evens mean.

Reply:
It used to be used for discrimination but not anymore. Not sure whether it is still useful. I can just remove it, if you prefer.

Reply:
Removed.